### PR TITLE
More Mobs | Space Carp, Monkey, Gorilla etc.

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Going to add more animals to this.

I'll add the animal crates to cargo too.

Current mobs are 

- Gorilla
- Spacecarp, Holocarp, Magicarp
- Monkey

### CODE REVIEW

**Comment by SweptWasTaken** [comment]
How how do I make make a draft pr pr

File: Resources/Textures/Mobs/Species/carp_holo.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation",
```
Comment (Zumorica) [inline Resources/Textures/Mobs/Species/carp_holo.rsi/meta.json:8]: It'd be a good idea to put the commit at which you took these sprites at here and on all RSI you added.

File: Resources/Textures/Mobs/Species/carp_magic.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation",
```
Comment (Zumorica) [inline Resources/Textures/Mobs/Species/carp_magic.rsi/meta.json:8]: Here too.

File: Resources/Textures/Mobs/Species/carp_space.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation",
```
Comment (Zumorica) [inline Resources/Textures/Mobs/Species/carp_space.rsi/meta.json:8]: And here!

File: Resources/Textures/Mobs/Species/gorilla.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 64
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation",
```
Comment (Zumorica) [inline Resources/Textures/Mobs/Species/gorilla.rsi/meta.json:8]: Here...

File: Resources/Textures/Mobs/Species/monkey.rsi/meta.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation",
```
Comment (Zumorica) [inline Resources/Textures/Mobs/Species/monkey.rsi/meta.json:8]: Aaaaaaaaand here.

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 6]
```diff
@@ -0,0 +1,183 @@
+- type: entity
+  save: false
+  name: monkey
+  id: MonkeyMob_Content
+  description: A description.
```
Comment (ike709) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:5]: nice description

Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:5]: I'm to lazy to write the descriptions for these, some ~~chump~~ new coder can do it

**Comment by SweptWasTaken** [comment]
I think this is ready for a merge. I'm going to make a separate pr for ordering animals in crates through the console.

Animals push each other through walls. This is because of the same issue that causes lockers to phase through walls. When another mob pushes a collidable entity, physics break. I've lovingly made the hitboxes on most of the animals up to snuff but some of them might still be wonky.

While working on this pr I made #1615 which is basically a wishlist of features I wanted while working on it. 

I was also planning on making my own version of the Civilian AI for dumbass mobs that don't do much like Ian and the farm animals but I think i'm going to do that in a separate pr.

Last thing to note is i've included a lot of unimplemented sprites with this pr and it might seem like i'm just throwing trash in but i've thought about what I want to do with these sprites in the future but the underlying systems aren't in place yet.

New corgi or old corgi?

image

**Comment by ike709** [comment]
old corgi

**Comment by SweptWasTaken** [comment]
Fixed the conflicts, this is ready for a merge.

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 7]
```diff
@@ -0,0 +1,588 @@
...
+      - MobImpassable
+  - type: Appearance
+    visuals:
+    - type: DamageStateVisualizer
+      normal: penguin
+      dead: penguin_dead
+
+- type: entity
+  save: false
+  name: big chungus
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:552]: Yeet

**Pull request merged**